bool initialized_
int64 data